system_tests/bigtable.py

```diff
@@ -93,3 +93,59 @@ def test_list_clusters(self):
             cluster_existence = (cluster in EXISTING_CLUSTERS or
                                  cluster == Config.CLUSTER)
             self.assertTrue(cluster_existence)
+
+    def test_reload(self):
+        # Use same arguments as Config.CLUSTER (created in `setUpModule`).
+        cluster = Config.CLIENT.cluster(CENTRAL_1C_ZONE, CLUSTER_ID)
+        # Make sure metadata unset before reloading.
+        cluster.display_name = None
+        cluster.serve_nodes = None
+
+        cluster.reload()
+        self.assertEqual(cluster.display_name, Config.CLUSTER.display_name)
+        self.assertEqual(cluster.serve_nodes, Config.CLUSTER.serve_nodes)
+
+    def test_create_cluster(self):
+        cluster_id = '%s-a' % (CLUSTER_ID,)
+        cluster = Config.CLIENT.cluster(CENTRAL_1C_ZONE, cluster_id)
+        operation = cluster.create()
+        # Make sure this cluster gets deleted after the test case.
+        self.clusters_to_delete.append(cluster)
+
+        # We want to make sure the operation completes.
```

Comment:
I meant something more along the lines of https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/bigquery/api/async_query.py#L57

but with a timeout or max number of tries, of course.